aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -147,12 +146,6 @@ module aptos_framework::genesis {
         // Give TransactionFee BurnCapability<AptosCoin> so it can burn gas.
         transaction_fee::store_aptos_coin_burn_cap(&aptos_framework_account, burn_cap);
 
-        // Pad the event counter for the Root account to match DPN. This
```

Comment:
Why did we have this in the first place and why is it safe to remove now?

Reply:
it's for backwards compatibility, the magic number in new_block_event_key, new_epoch_event_key

Reply:
And we no longer need this anymore? Technically if we never have a long-lived network, why do we need padding?

Reply:
the code was copied from a diem file I think, so finally get a chance to clean it up